airflow/models/slamiss.py

```diff
@@ -44,3 +49,7 @@ class SlaMiss(Base):
 
     def __repr__(self):
         return str((self.dag_id, self.task_id, self.execution_date.isoformat()))
+
+    @property
```

Comment:
Should we only merge it for 2.11.0 ? 
Or do want to add it to 2.10.4 ? I marked it provisionally as 2.11.0. 

Note that we do not have currently a way to merge 2.11.0-only changes - so, unless we introduce a solution for that - it will have to wait until we stop preparing 2.10.* patchlevel releases @uranusjr @ephraimbuddy @utkarsharma2

Reply:
We could also potentially have `v2-11-test` branch at some point of time (this is mostly the matter of how many changes like that we are going to have). Theorethically, there is nothing wrong with having both branches created and having Pull Requests to v2-11-test. We could then later rebase v2-11-test on top of v2-10-test. - but that basically means that someone who merges changes now would have to resolve the conflicts if that happens - or maybe put the whole weitght of this resolving on the shoulders of release manager - both are not too good ideas, likely keeping such changes as DRAFT PR and rebasing them individually/merging PRs later is a better idea - as it puts the burden on solving the issue on the one who is author (i.e. knows best what the intention and context was).

Reply:
> Likely keeping such changes as DRAFT PR and rebasing them individually/merging PRs later is a better idea - as it puts the burden on solving the issue on the one who is author (i.e. knows best what the intention and context was).

I prefer this